androidApp/src/main/java/com/mbta/tid/mbta_app/android/ContentView.kt

```diff
@@ -62,6 +66,16 @@ fun ContentView(
         (socket as? PhoenixSocketWrapper)?.attachLogging()
         onDispose { socket.detach() }
     }
+
+    if (!pendingOnboarding.isNullOrEmpty() && !onboardingJustCompleted) {
+        OnboardingPage(
+            pendingOnboarding,
+            onFinish = { onboardingJustCompleted = true },
```

Comment:
On iOS, this handler sets the UI state directly
https://github.com/mbta/mobile_app/blob/673e762e7dc4819b7f24a0a9855503713a20de79/iosApp/iosApp/ContentView.swift#L91-L93
but our Android ViewModel-driven state pattern does not really lend itself well to either overwriting the cached state or manually refreshing the data on some external event. This approach is not elegant, but it is direct and fits within the existing architecture.